### PR TITLE
Update hypertension newsletter with header image & change to footer

### PR DESCRIPTION
**Pull request by jwade1327**
Screen Shot 2021-08-30 at 8 39 52 AM

### CODE REVIEW

**Comment by jwade1327** [comment]
[Phase 1: AHA - Hypertension eNewsletter Initial Setup](https://trello.com/c/gN9fu4CT/165-phase-1-aha-hypertension-enewsletter-initial-setup)

**Comment by jwade1327** [comment]
Screen Shot 2021-08-30 at 9 43 19 AM

File: packages/common/components/blocks/footer-aha.marko [hunk 1]
```diff
@@ -83,44 +91,66 @@ $ const taglineStyle = {
...
-            </tr>
-            <common-table-spacer-element height="16" />
-            <tr>
-              <td align="left" valign="top" style=standardStyle>
-                ${unsubscribe}
-                <a href="@{confirmunsubscribelink}@" style=standardUnderlineStyle>
-                click here.
-                </a>
+            <if(brand)>
+              <td align="right" valign="top" style=brandStyle>
```
Comment (B77Mills) [inline packages/common/components/blocks/footer-aha.marko:95]: @jwade1327  This should be wrapped in a <tr></tr>

**Review by B77Mills — CHANGES_REQUESTED** [review]

File: packages/common/components/blocks/footer-aha.marko (hunk 1 above)
Comment (jwade1327) [reply packages/common/components/blocks/footer-aha.marko]: @B77Mills change has been made!

**Pull request merged**